.circleci/config.yml

```diff
@@ -18,14 +18,26 @@ workflows:
               ignore:
                 - master
       - codacy/shell:
-          name: build-and-publish
+          name: build-and-publish-2_12
           context: CodacyAWS
           cmd: |
             export VERSION="$(cat .version)"
             # export SCALA_VERSION="2.12.9"
-            ./gradlew build publishToSonatype getStagingProfile closeAndReleaseRepository
+            ./gradlew build publishToSonatype getStagingProfile closeAndReleaseRepository --stacktrace
+          requires:
+            - codacy/checkout_and_version
+            - build-and-publish-2_11
```

Comment:
Since this one only runs after the one below can we change the order?

Reply:
explained bellow

Reply:
I meant moving the code of the 2_11 above the 2_12, so it is consistent with what will happen in the CI